source/NVDAObjects/UIA/__init__.py

```diff
@@ -853,7 +853,11 @@ def findOverlayClasses(self,clsList):
 		# Windows 8.x toast, although a form of tool tip, is covered separately.
 		elif UIAControlType==UIAHandler.UIA_ToolTipControlTypeId:
 			clsList.append(ToolTip)
-		elif self.UIAElement.cachedFrameworkID in ("InternetExplorer","MicrosoftEdge"):
+		elif(
+			self.UIAElement.cachedFrameworkID in ("InternetExplorer", "MicrosoftEdge")

```

Comment:
I'm pretty confident that Internet Explorer can be removed here, as that would only be true for the mshtml implementation, not edgeHTML. And, we would want this fix for any mshtml embedding, not just the Internet Explorer application.

Reply:
Removing InternetExplorer from there would break Edge on older builds of Windows 10 (it is used at least on 1507, 1511 and possibly Anniversary Update) as an FrameworkID. The fact that UIA is used for Internet Explorer controls on Windows 7 is however strange, because this is not occurring on Windows 10, I am not sure why.
Would it be acceptable to simply add Internet Explorer_Server to the bad UIA class names when running on anything lower than Windows 10?